homeassistant/components/sensibo/strings.json

```
@@ -485,8 +485,8 @@
       }
     },
     "enable_climate_react": {
-      "name": "Enable climate react",
-      "description": "Enables and configures climate react.",
+      "name": "Enable Climate React",
+      "description": "Enables and configures the Climate React mode.",
```

```suggestion
      "description": "Enables and configures Climate React.",
```
It's not a mode in itself.